### PR TITLE
Remove the nodes command-line client

### PR DESCRIPTION
**Pull request by lstein**
## What type of PR is this? (check all applicable)

- [X] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Optimization
- [X] Documentation Update
- [ ] Community Node Submission

## Have you discussed this change with the InvokeAI team?
- [X] Yes
- [ ] No, because:

      
## Have you updated all relevant documentation?
- [X] Yes
- [ ] No

## Description

This PR:

1. Removes the non-functional and unmaintained nodes command line client
2. Removes the CLI command from the launcher
3. Moves all CLI documentation into `deprecated`
4. Fixes a bunch of unrelated broken documentation links
5. Moves CODE_OF_CONDUCT.md into `docs`

## Related Tickets & Documents

- Related Issue #
- Closes #

## Added/updated tests?

- [ ] Yes
- [X] N/A

### CODE REVIEW

**Review by hipsterusername — APPROVED** [review]
RIP

File: pyproject.toml [hunk 1]
```diff
@@ -125,7 +125,7 @@ dependencies = [
 # shortcut commands to start cli and web
 # "invokeai --web" will launch the web interface
 # "invokeai" will launch the CLI
```
Comment (psychedelicious) [inline pyproject.toml:127]:
```suggestion
# shortcut commands to start web ui
# "invokeai --web" will launch the web interface
```

**Review by psychedelicious — APPROVED** [review]
Farewell CLI. Hope to see you again in the future.

**Review by Millu — APPROVED** [review]

**Comment by blessedcoolant** [comment]
RIP. Literally what got me to lstein's repo back then.

**Review by blessedcoolant — APPROVED** [review]

**Pull request merged**